sympy/functions/elementary/complexes.py

```diff
@@ -486,7 +486,8 @@ def eval(cls, arg):
                 if any(a.is_infinite for a in arg.as_real_imag()):
                     return S.Infinity
             if arg.is_real is None and arg.is_imaginary is None:
-                if all(a.is_real or a.is_imaginary or (S.ImaginaryUnit*a).is_real for a in arg.args):
+                if all(a.is_real or a.is_imaginary
+                       or (S.ImaginaryUnit*a).is_real for a in arg.args):
```

Comment:
This obscures the last person who touched the line with a semantic change.
Overly long lines aren't nice, but I do not think that fixing them is worth the price.